.github/workflows/ci.yml

```diff
@@ -44,8 +45,8 @@ jobs:
           pip install flake8 black
       - name: lint
         run: |
-          black --check ff2mpv.py check-config-win.py
-          flake8 ff2mpv.py check-config-win.py
+          black --check ff2mpv.py helper_win.py
```

Comment:
Instead of creating a brand new `helper_win.py`, let's just replace the current `check-config-win.py` with this one. That makes the Git history simpler and requires fewer doc updates.